engine/src/main/java/io/zeebe/engine/processing/message/MessageEventProcessors.java

```diff
@@ -70,8 +70,8 @@ public static void addMessageProcessors(
         .onCommand(
             ValueType.MESSAGE_SUBSCRIPTION,
             MessageSubscriptionIntent.REJECT,
-            new RejectMessageCorrelationProcessor(
-                messageState, subscriptionState, subscriptionCommandSender))
+            new MessageSubscriptionRejectProcessor(
```

Comment:
Currently we have not the concept for processing command rejections right? That is why we have a separate command for it?

Reply:
Yes, rejections are not processed. 
Here, the situation is slightly different. The message was not correlated and some locks need to be rolled back. But the correlation may happen on a different partition. If it would be on the same partition then we could reject the correlation command as normal.